PayPalMessaging/src/main/java/com/braintreepayments/api/PayPalMessagingRequest.kt

```diff
@@ -0,0 +1,26 @@
+package com.braintreepayments.api
+
+/**
+ * Used to initialize a [PayPalMessagingView]
+ * This feature is currently only supported for buyers located in the US. For merchants domiciled
+ * outside of the US please set the [buyerCountry] to display messaging to US based buyers.
+ * Note: **This module is in beta. It's public API may change or be removed in future releases.**
+ *
+ * @property amount Price expressed in cents amount based on the current context
+ * (i.e. individual product price vs total cart price)
+ * @property placement Message screen location (e.g. product, cart, home)
+ * @property offerType Preferred message offer to display
+ * @property buyerCountry Consumer's country (Integrations must be approved by PayPal to use this option)
+ * @property logoType Logo type option for a PayPal Message. Defaults to [PayPalMessagingLogoType.INLINE]
+ * @property textAlignment Text alignment option for a PayPal Message. Defaults to [PayPalMessagingTextAlignment.RIGHT]
+ * @property color Text and logo color option for a PayPal Message. Defaults to [PayPalMessagingColor.BLACK]
+ */
+data class PayPalMessagingRequest(
```

Comment:
```suggestion
data class PayPalMessagingConfig(
```
🧶 : It feels more like the merchant is configuring a view, and the messaging SDK is making network requests under the hood. `PayPalMessagingOptions` could be another alternative name.

`Request` is a well known suffix in the SDK though. Take-or-leave.

Reply:
Yeah, I think request makes sense from a consistency point of view across the other modules